src/qibo/backends/__init__.py

```diff
@@ -1,35 +1,63 @@
 import os
+from pkgutil import iter_modules
 from qibo import config
 from qibo.config import raise_error, log, warnings
 from qibo.backends.numpy import NumpyDefaultEinsumBackend, NumpyMatmulEinsumBackend
 from qibo.backends.tensorflow import TensorflowCustomBackend, TensorflowDefaultEinsumBackend, TensorflowMatmulEinsumBackend
```

Comment:
We can could also move these imports under the `if _check_availability("tensorflow")` part below (line 29) to keep the rest of the file clean of backends that are not available.